src/components/Dashboard/Health/HealthItem.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from 'patternfly-react';
+
+export const HealthItem = ({ icon, classname, message }) => (
+  <React.Fragment>
+    <div className="kubevirt-ocs-health__icon">
```

Comment:
all styles are refering to `ocs`. We should use `kubevirt-dashboard-health` styles